src/test/clip.t.sol

```diff
@@ -692,4 +692,80 @@ contract ClipperTest is DSTest {
         uint256 lotReturn = 30 ether - expectedGem;         // lot - loaf.tab / max = 15
         assertEq(vat.gem(ilk, me), 960 ether + lotReturn);  // Collateral returned (10 WAD)
     } 
+
+    function auctionResetSetup(uint256 tau) internal {
+        LinearDecrease calc = new LinearDecrease();
+        calc.file(bytes32("tau"), tau);     // tau hours till zero is reached (used to test tail)  
+
+        clip.file("buf",  ray(1.25 ether)); // 25% Initial price buffer
+        clip.file("dust", rad(20   ether)); // $20 dust
+        clip.file("calc", address(calc));   // File price contract
+        clip.file("cusp", ray(0.5 ether));  // 50% drop before reset
+        clip.file("tail", 3600);            // 1 hour before reset
+
+        assertEq(clip.kicks(), 0);
+        dog.bark(ilk, me);
+        assertEq(clip.kicks(), 1);
+    }
+
+    function try_redo(uint256 id) internal returns (bool ok) {
+        string memory sig = "redo(uint256)";
+        (ok,) = address(clip).call(abi.encodeWithSignature(sig, id));
+    }
+
+    function test_auction_reset_tail() public {
+        auctionResetSetup(10 hours); // 10 hours till zero is reached (used to test tail) 
+
+        pip.poke(bytes32(uint256(3 ether))); // Spot = $1.50 (update price before reset is called)
+
+        (,,,, uint96 tic_before, uint256 top_before) = clip.sales(1);
```

Comment:
I hate to be that guy...but we generally `camelCase` variable names (see e.g. `startTime`, and other values in this file), and I'd like to try to maintain that style, or at the very least have each source file be self-consistent in its formatting conventions. Mind changing `tic_before` ==> `ticBefore`, etc?

Reply:
Yeah I can do that 👍